docs/diagnostics/PX1088.md

```diff
@@ -0,0 +1,44 @@
+# PX1088
+This document describes the PX1088 diagnostic.
+
+## Summary
+
+| Code   | Short Description                                                                                                                                          | Type                           | Code Fix    | 
+| ------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------ | ----------- | 
+| PX1088 | Processing delegates cannot use the data views from processing graphs, except for the data views of the `PXFilter`, `PXProcessingBase`, and `PXSetup` types. | Warning (Level 2: Production Quality) | Unavailable |
+
+## Diagnostic Description
+
+Processing delegates cannot use the data views from processing graphs except for the data views of the `PXFilter`, `PXProcessingBase`, and `PXSetup` types.
+
+To prevent the warning from occurring, you should remove from the processing delegate the code that uses the data views of types other than `PXFilter`, `PXProcessingBase`, or `PXSetup`.
+
+## Example of Code That Results in the Warning
+
+```C#
+public class DocProcess : PXGraph<DocProcess>
+{
+    public PXProcessing<
+        SOShipment,
+        Where2<
+            Where<SOShipmentExtension.aSNSent, Equal<False>,
+                Or<SOShipmentExtension.aSNSent, IsNull>>,
+            And2<
+                Where<SOShipmentExtension.readyToSend, Equal<True>>,
+                And<SOShipmentExtension.shipmentTypePkey, IsNotNull>>>>
+        Shipments;
+    public PXSetup<CustomerSettings> CustomerSettings;
+    public PXSelect<SOShipment> AllShipments;
+
+    public void SomeMethod()
```

Comment:
It is not clear that `SomeMethod()` is a processing delegate.
I think that the constructor with `Shipments.SetProcessDelegate(SomeMethod)` should be added.